blogs/index-cardinality-detection/README.md

```diff
@@ -0,0 +1,112 @@
+# Detecting low cardinality indexes for DocumentDB Performance 
+
+Amazon DocumentDB uses a B-tree data structure for its indexes. A B-tree index uses a hierarchical structure that stores data in its nodes in a sorted order. B-Trees are excellent data structure for fast retrievals when index cardinality is high. As a rule of thumb Amazon DocumentDB indexes should have less than 1% selectivity. This script accepts documentdb cluster endpoint as a parameter and generates a report across all the databases and collections for indexes lower than threshold ( Default 1% ) provided. 
+
+### Supported Parameters 
+
+### Requirements 
+* Python 3.9+ installed 
```

Comment:
3.9+ or 3.x ?

Reply:
I tested with 3.9 but dont see anything that wont work with 3.x. I will change it. see if you get a chance to test with lower versions.